calcom/cal.com
# refactor: [CAL-2204] RFC: mandatory email reminder for attendees with @gmail.com

## What does this PR do?
This pr schedules email 1 hr before the event starts  for attendees which uses gmail account if not already setup by the creator from workflows.If the booked event got cancelled , then the reminder will also get cancelled. In case of event which requires confirmation, email  will only be scheduled  when the event is confirmed. 

Fixes #10247 


## Type of change

## How should this be tested?
Make sure you had sendgrid Setup, then create meeting from eventtypes and then book the meeting using one of the attendee mail as gmail.com.
